app/views/articles/search.html.erb

```diff
@@ -10,6 +10,17 @@
   <%= render "articles/search/sidebar" %>
   <div class="articles-list" id="articles-list">
     <div id="banner-section"></div>
+    <% if (params[:q]&.downcase == "job" || params[:q]&.downcase == "jobs") && SiteConfig.display_jobs_banner %>
+      <div class="crayons-notice crayons-notice--info mb-3 fs-base">Interested in joining our team? Explore our <a href=<%= SiteConfig.jobs_url %>>open roles</a>.</div>
+    <% end %>
```

Comment:
I think this part should be removed. see `app/views/articles/_search`. I was added via JS in #8303